src/main/antlr4/Cryptator.g4

```diff
@@ -38,6 +42,8 @@ addORsub : '+' | sub;
 
 sub : '-';
 
+logicOperator : ';';
```

Comment:
'and' to keep the same naming scheme ?

Reply:
²yes I think 'and' is better I change it